### DISCUSSION TITLE
Adhoc SSH

### DISCUSSION THREAD

**Discussion by neonvoidx** [kovidgoyal/kitty#8132] [Q&A]
https://wezfurlong.org/wezterm/multiplexing.html#ssh-domains
Is there a way to adhoc ssh, similar to wezterm, where my server can have a kitty daemon of sorts i can connect to?
Or if there is a better way to do this, essentially i want to create a socket on server I can connect to and disconnect from at any time, reconnecting without losing where I was. I know I can do this with wezterm or tmux but I don't use either

**Comment by kovidgoyal** [comment]
No, in kitty you use the ssh kitten instead.

On Tue, Dec 17, 2024 at 05:43:10PM -0800, Jacob Reed wrote:
> https://wezfurlong.org/wezterm/multiplexing.html#ssh-domains
> Is there wa way to adhoc ssh, similar to wezterm, where my server can have a kitty daemon of sorts i can connect to?
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/8132
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>

--
_____________________________________

Dr. Kovid Goyal
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by neonvoidx** [reply]
I am using ssh kitten but that doesnt keep a persistent session that i can reconnect to, or does it?

**Comment by kovidgoyal** [comment]
No it doesnt.
